manifests/a/Anaconda/Miniconda3/py39_4.10.3/Anaconda.Miniconda3.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.0.0/manifest.defaultLocale.1.0.0.json
+PackageIdentifier: Anaconda.Miniconda3
+Moniker: miniconda3
+PackageVersion: py39_4.10.3
+PackageLocale: en-US
+Publisher: Anaconda
```

Comment:
```suggestion
Publisher: Anaconda, Inc.
```

Reply:
Older version of package has same field for publisher. The old version is also installed before the newer versions prepended with py*.
https://github.com/microsoft/winget-pkgs/blob/master/manifests/a/Anaconda/Miniconda3/4.8.3/Anaconda.Miniconda3.yaml

Reply:
I see, @ItzLevvie corrected py39_4.9.2 which fixed the matching but left 4.8.3 which broke the version selection..
Would you mind updating or deleting that version?

Reply:
Alright, changes made